includes/class-custom-post-type-fund.php

```diff
@@ -43,8 +44,65 @@ function custom_post_types() {
 			'hierarchical' => false,
 			'taxonomies' => array( 'idonate_priorities', 'idonate_colleges', 'idonate_campuses', 'idonate_programs' ),
 			'supports' => array( 'title', 'custom-fields' ),
+			'capability_type' => array( 'idonate_fund' , 'idonate_funds' ),
+			'map_meta_cap' => true,
 		);
 
 		register_post_type( 'idonate_fund', $args );
 	}
+
+	# Give Administrators All Fund Editing Capabilities
+	function add_idonate_fund_caps_to_admin() {
+		$caps = array(
+			'read',
+			'read_idonate_fund',
+			'read_private_idonate_funds',
+			'edit_idonate_funds',
+			'edit_private_idonate_funds',
+			'edit_published_idonate_funds',
+			'edit_others_idonate_funds',
+			'publish_idonate_funds',
+			'delete_idonate_funds',
+			'delete_private_idonate_funds',
+			'delete_published_idonate_funds',
+			'delete_others_idonate_funds',
+		);
+
+		$roles = array(
+			get_role( 'administrator' ),
+		);
+
+		foreach ( $roles as $role ) {
+			foreach ( $caps as $cap ) {
+				$role->add_cap( $cap );
+			}
+		}
+	}
+
+	# Give other roles Fund editing Capabilities
+	function add_idonate_fund_caps_to_non_admin_roles() {
+		add_role( 'fund_editor', 'Fund Editor' );
+
+		# Everyone gets these capabilities:
+		$caps = array(
+			'read_idonate_fund',
+			'read_private_idonate_funds',
+			'edit_idonate_funds',
+			'edit_private_idonate_funds',
+			'edit_published_idonate_funds',
+			'edit_others_idonate_funds',
+			'publish_idonate_funds',
+			'delete_idonate_funds',
+		);
+
+		$roles = array(
+			get_role( 'fund_editor' ),
```

Comment:
@philcable I have updated the comments to reflect the actual intentions of the fund editor role and it's capabilities and simplified it as you suggested. Can you please #reviewmerge again?